pkgs/tools/networking/ogen/default.nix

```
@@ -0,0 +1,25 @@
+{
+  pkgs,
+  lib,
+}:
```

You should import the dependencies directly, not use them via `pkgs`:

```suggestion
{ buildGoModule
, fetchFromGitHub
, lib
,
}:
```